### DISCUSSION TITLE
how to donate?

### DISCUSSION THREAD

**Discussion by boogerlad** [agentm/project-m36#334] [General]
I can see this finally getting us out of the local optimum of sql databases. I'd love to donate some monero to fund development! (not shilling - genuinely think it's great)

**Comment by agentm** [comment]
That's very generous! Currently, we have no means of receiving donations, since no one has offered one before. Is there a donation system you would recommend?

**Reply by boogerlad** [reply]
[Github Sponsors](https://github.com/sponsors) seems popular, but personally I would prefer [monero](https://featherwallet.org/) due to simplicity (just paste a donation address into the readme, for example), and privacy. You can find out more about the cryptocurrency [here](https://www.getmonero.org/).
